README.md

```diff
@@ -10,9 +10,10 @@ language: C++
 This repository contains EDJX's sample serverless functions written in C++.
 Each sample also contains a README, which has a detailed explanation about
 the specifics of the Serverless Function. These sample functions can be
-deployed to EDJX network using EDJX CLI. More details on how to deploy via
+built and 'deployed to EDJX network' using EDJX CLI. More details on how to deploy via
```

Comment:
The introductory paragraph was copied from the Rust repo. Your update makes sense. Do the apostrophes added around 'deployed to EDJX network' have any meaning?

Reply:
I thought that would make sense.. apostrophes. But after thought.. felt not needed.